<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta name="viewport" content="width=device-width, initial-scale=1" />
<meta property="og:title" content="Chapter 7 Additional materials and session information | R for Mass Spectrometry" />
<meta property="og:type" content="book" />




<meta name="author" content="Laurent Gatto, Sebastian Gibb, Johannes Rainer" />

<meta name="date" content="2023-06-01" />


<meta name="description" content="Chapter 7 Additional materials and session information | R for Mass Spectrometry">

<title>Chapter 7 Additional materials and session information | R for Mass Spectrometry</title>

<link href="libs/tufte-css-2015.12.29/tufte.css" rel="stylesheet" />
<link href="libs/tufte-css-2015.12.29/envisioned.css" rel="stylesheet" />
<link href="libs/msmb-css-0/msmb.css" rel="stylesheet" />
<script>
function toggle_visibility(id1, id2) {
var e = document.getElementById(id1);
var f = document.getElementById(id2);

e.style.display = ((e.style.display!='none') ? 'none' : 'block');

if(f.classList.contains('fa-plus-square')) {
    f.classList.add('fa-minus-square')
    f.classList.remove('fa-plus-square')
} else {
    f.classList.add('fa-plus-square')
    f.classList.remove('fa-minus-square')
}

}
</script>
<script>
function copy_link(id) {
  var dummy = document.createElement('input'),
  text = window.location.href.split(/[?#]/)[0] + '#' + id;
  document.body.appendChild(dummy);
  dummy.value = text;
  dummy.select();
  document.execCommand('copy');
  document.body.removeChild(dummy);
  
  var tooltip = document.getElementById(id + '-tooltip');
  tooltip.innerHTML = 'Copied!';
}

function reset_tooltip(id) {
  var tooltip = document.getElementById(id);
  tooltip.innerHTML = 'Copy link';
}
</script>
<script src="libs/accessible-code-block-0.0.1/empty-anchor.js"></script>


<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
div.sourceCode { margin: 1em 0; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {  background-color: #f8f8f8; }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ef2929; } /* Alert */
code span.an { color: #8f5902; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #c4a000; } /* Attribute */
code span.bn { color: #0000cf; } /* BaseN */
code span.cf { color: #204a87; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4e9a06; } /* Char */
code span.cn { color: #000000; } /* Constant */
code span.co { color: #8f5902; font-style: italic; } /* Comment */
code span.cv { color: #8f5902; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #8f5902; font-weight: bold; font-style: italic; } /* Documentation */
code span.dt { color: #204a87; } /* DataType */
code span.dv { color: #0000cf; } /* DecVal */
code span.er { color: #a40000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #0000cf; } /* Float */
code span.fu { color: #000000; } /* Function */
code span.im { } /* Import */
code span.in { color: #8f5902; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #204a87; font-weight: bold; } /* Keyword */
code span.op { color: #ce5c00; font-weight: bold; } /* Operator */
code span.ot { color: #8f5902; } /* Other */
code span.pp { color: #8f5902; font-style: italic; } /* Preprocessor */
code span.sc { color: #000000; } /* SpecialChar */
code span.ss { color: #4e9a06; } /* SpecialString */
code span.st { color: #4e9a06; } /* String */
code span.va { color: #000000; } /* Variable */
code span.vs { color: #4e9a06; } /* VerbatimString */
code span.wa { color: #8f5902; font-weight: bold; font-style: italic; } /* Warning */
</style>



<link rel="stylesheet" href="style.css" type="text/css" />

</head>

<body>



<div class="row">
<div class="col-sm-12">
<div id="TOC">
<ul class="navbar">
<li class="msmb"><p class="title">R for Mass Spectrometry<p><p class="author">Laurent Gatto, Sebastian Gibb, Johannes Rainer</p>
<li class="dropdown" style="float:right">
<a href="javascript:void(0)" class="dropbtn">&#x25BE; Chapters</a>
<div class="dropdown-content">
<a href="index.html"><span class="toc-section-number">1</span> Preamble</a>
<a href="sec-msintro.html"><span class="toc-section-number">2</span> Introduction</a>
<a href="sec-raw.html"><span class="toc-section-number">3</span> Raw MS data</a>
<a href="sec-id.html"><span class="toc-section-number">4</span> Identification data</a>
<a href="sec-quant.html"><span class="toc-section-number">5</span> Quantitative data</a>
<a href="sec-anx.html"><span class="toc-section-number">6</span> Annex</a>
<a id="active-page" href="sec-si.html"><span class="toc-section-number">7</span> Additional materials and session information</a><ul class="toc-sections">
<li class="toc"><a href="#additional-materials"> Additional materials</a></li>
<li class="toc"><a href="#session-information"> Session information</a></li>
</ul>
</div>
</li>
</ul>
</div>
</div>
</div>
<div class="row">
<div class="col-sm-12">
<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN" "http://www.w3.org/TR/REC-html40/loose.dtd">
<html><body><div id="sec-si" class="section level1" number="7">
<h1>
<span class="header-section-number">Chapter 7</span> Additional materials and session information</h1>
<div id="additional-materials" class="section level2" number="7.1">
<h2>
<span class="header-section-number">7.1</span> Additional materials<div class="tooltip"><button class="internal-link-btn" onclick="copy_link('additional-materials')" onmouseout="reset_tooltip('additional-materials-tooltip')"><span class="tooltiptext" id="additional-materials-tooltip">Copy link</span><i class="fa fa-link"></i></button></div>
</h2>
<ul>
<li><p>The <a href="https://lgatto.github.io/QFeaturesScpWorkshop2021/">Single-cell proteomics data analysis using <code>QFeatures</code> and
<code>scp</code></a> workshop
is provided as two vignettes. The first one provides a general
introduction to the <code>QFeatures</code> class in the general context of mass
spectrometry-based proteomics data manipulation. The second vignette
focuses on single-cell application and introduces the <code>scp</code> package
<span class="citation">(Vanderaa and Gatto <label for="tufte-mn-17" class="margin-toggle">⊕</label><input type="checkbox" id="tufte-mn-17" class="margin-toggle">2021<span class="marginnote">Vanderaa, Christophe, and Laurent Gatto. 2021. “Replication of Single-Cell Proteomics Data Reveals Important Computational Challenges.” <em>Expert Rev. Proteomics</em>, October.</span>)</span> as an extension of <code>QFeatures</code>. This second
vignette also provides exercises that give the attendee the
opportunity to apply the learned concepts to reproduce a published
analysis on a subset of a real data set.</p></li>
<li><p>The <a href="https://jorainer.github.io/SpectraTutorials/">Seamless Integration of Mass Spectrometry Data from Different
Sources</a> workshop
explains the <code>Spectra</code> package and shows how this new infrastructure
can be used to represent and analyze Mass Spectrometry (MS)
data. The use of different data type and format specific backends
guarantees an easy expandability of the infrastructure. This is
illustrated in this tutorial on a simple use case, in which MS/MS
(a.k.a. MS2) spectra, measured by an MS instrument, are compared and
matched against a public spectral database containing public domain
reference MS2 spectra.</p></li>
<li><p>A tutorial presenting <a href="https://jorainer.github.io/MetaboAnnotationTutorials/">Use Cases and Examples for Annotation of
Untargeted Metabolomics
Data</a> using
the <code>MetaboAnnotation</code> and <code>MetaboCoreUtils</code> packages
<span class="citation">(Rainer et al. <label for="tufte-mn-18" class="margin-toggle">⊕</label><input type="checkbox" id="tufte-mn-18" class="margin-toggle">2022<span class="marginnote">Rainer, Johannes, Andrea Vicini, Liesa Salzer, Jan Stanstrup, Josep M Badia, Steffen Neumann, Michael A Stravs, et al. 2022. “A Modular and Expandable Ecosystem for Metabolomics Data Annotation in R.” <em>Metabolites</em> 12 (2): 173.</span>)</span>.</p></li>
<li><p>A tutorial for <a href="https://jorainer.github.io/metabolomics2018/xcms-preprocessing.html">LC-MS pre-processing with
<code>xcms</code></a>
with examples on how to perform centroiding of profile-mode MS data
in <code>R</code> and how to define data set-specific parameters for
chromatographic peak detection, alignment and correspondence of
untargeted (LC-MS-based) metabolomics data.</p></li>
</ul>
</div>
<div id="session-information" class="section level2" number="7.2">
<h2>
<span class="header-section-number">7.2</span> Session information<div class="tooltip"><button class="internal-link-btn" onclick="copy_link('session-information')" onmouseout="reset_tooltip('session-information-tooltip')"><span class="tooltiptext" id="session-information-tooltip">Copy link</span><i class="fa fa-link"></i></button></div>
</h2>
<p>The following packages have been used to generate this document.</p>
<div class="sourceCode" id="cb470"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb470-1"><a href="sec-si.html#cb470-1" aria-hidden="true"></a><span class="kw">sessionInfo</span>()</span></code></pre></div>
<pre><code>## R version 4.3.0 (2023-04-21)
## Platform: x86_64-pc-linux-gnu (64-bit)
## Running under: Ubuntu 22.04.2 LTS
## 
## Matrix products: default
## BLAS:   /usr/lib/x86_64-linux-gnu/atlas/libblas.so.3.10.3 
## LAPACK: /usr/lib/x86_64-linux-gnu/atlas/liblapack.so.3.10.3;  LAPACK version 3.10.0
## 
## locale:
##  [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C              
##  [3] LC_TIME=en_GB.UTF-8        LC_COLLATE=en_US.UTF-8    
##  [5] LC_MONETARY=en_GB.UTF-8    LC_MESSAGES=en_US.UTF-8   
##  [7] LC_PAPER=en_GB.UTF-8       LC_NAME=C                 
##  [9] LC_ADDRESS=C               LC_TELEPHONE=C            
## [11] LC_MEASUREMENT=en_GB.UTF-8 LC_IDENTIFICATION=C       
## 
## time zone: Europe/Brussels
## tzcode source: system (glibc)
## 
## attached base packages:
## [1] stats4    stats     graphics  grDevices utils     datasets  methods  
## [8] base     
## 
## other attached packages:
##  [1] mzID_1.38.0                 patchwork_1.1.2            
##  [3] factoextra_1.0.7            gplots_3.1.3               
##  [5] limma_3.56.1                lubridate_1.9.2            
##  [7] forcats_1.0.0               stringr_1.5.0              
##  [9] purrr_1.0.1                 readr_2.1.4                
## [11] tibble_3.2.1                tidyverse_2.0.0            
## [13] MSnID_1.34.0                tidyr_1.3.0                
## [15] ggplot2_3.4.2               dplyr_1.1.2                
## [17] magrittr_2.0.3              msdata_0.40.0              
## [19] rpx_2.8.0                   MsCoreUtils_1.12.0         
## [21] QFeatures_1.10.0            MultiAssayExperiment_1.26.0
## [23] SummarizedExperiment_1.30.1 Biobase_2.60.0             
## [25] GenomicRanges_1.52.0        GenomeInfoDb_1.36.0        
## [27] IRanges_2.34.0              MatrixGenerics_1.12.0      
## [29] matrixStats_0.63.0          Spectra_1.10.1             
## [31] ProtGenerics_1.32.0         BiocParallel_1.34.2        
## [33] S4Vectors_0.38.1            BiocGenerics_0.46.0        
## [35] mzR_2.34.0                  Rcpp_1.0.10                
## [37] BiocStyle_2.28.0           
## 
## loaded via a namespace (and not attached):
##   [1] later_1.3.1             bitops_1.0-7            filelock_1.0.2         
##   [4] R.oo_1.25.0             preprocessCore_1.62.1   XML_3.99-0.14          
##   [7] lifecycle_1.0.3         rstatix_0.7.2           doParallel_1.0.17      
##  [10] lattice_0.21-8          MASS_7.3-59             backports_1.4.1        
##  [13] sass_0.4.6              rmarkdown_2.21          jquerylib_0.1.4        
##  [16] yaml_2.3.7              httpuv_1.6.11           DBI_1.1.3              
##  [19] RColorBrewer_1.1-3      abind_1.4-5             zlibbioc_1.46.0        
##  [22] R.cache_0.16.0          R.utils_2.12.2          AnnotationFilter_1.24.0
##  [25] RCurl_1.98-1.12         rappdirs_0.3.3          GenomeInfoDbData_1.2.10
##  [28] ggrepel_0.9.3           pheatmap_1.0.12         MSnbase_2.26.0         
##  [31] ncdf4_1.21              codetools_0.2-19        DelayedArray_0.26.3    
##  [34] xml2_1.3.4              tidyselect_1.2.0        farver_2.1.1           
##  [37] BiocFileCache_2.8.0     jsonlite_1.8.4          ellipsis_0.3.2         
##  [40] iterators_1.0.14        foreach_1.5.2           tools_4.3.0            
##  [43] glue_1.6.2              BiocBaseUtils_1.2.0     xfun_0.39              
##  [46] withr_2.5.0             BiocManager_1.30.20     fastmap_1.1.1          
##  [49] rhdf5filters_1.12.1     fansi_1.0.4             caTools_1.18.2         
##  [52] digest_0.6.31           timechange_0.2.0        R6_2.5.1               
##  [55] mime_0.12               colorspace_2.1-0        gtools_3.9.4           
##  [58] RSQLite_2.3.1           R.methodsS3_1.8.2       utf8_1.2.3             
##  [61] generics_0.1.3          data.table_1.14.8       httr_1.4.6             
##  [64] S4Arrays_1.0.4          pkgconfig_2.0.3         gtable_0.3.3           
##  [67] blob_1.2.4              impute_1.74.1           XVector_0.40.0         
##  [70] htmltools_0.5.5         carData_3.0-5           bookdown_0.34.2        
##  [73] MALDIquant_1.22.1       clue_0.3-64             scales_1.2.1           
##  [76] png_0.1-8               knitr_1.43              rstudioapi_0.14        
##  [79] tzdb_0.4.0              reshape2_1.4.4          curl_5.0.0             
##  [82] cachem_1.0.8            rhdf5_2.44.0            BiocVersion_3.17.1     
##  [85] KernSmooth_2.23-20      parallel_4.3.0          AnnotationDbi_1.62.1   
##  [88] vsn_3.68.0              msmbstyle_0.0.19        pillar_1.9.0           
##  [91] grid_4.3.0              vctrs_0.6.2             pcaMethods_1.92.0      
##  [94] promises_1.2.0.1        ggpubr_0.6.0            car_3.1-2              
##  [97] dbplyr_2.3.2            xtable_1.8-4            cluster_2.1.4          
## [100] evaluate_0.21          
##  [ reached getOption("max.print") -- omitted 28 entries ]</code></pre>

</div>
</div></body></html>

<p style="text-align: center;">
<a href="sec-anx.html"><button class="btn btn-default">Previous</button></a>
</p>
<p class="build-date">Page built: 
2023-06-01
 using 
R version 4.3.0 (2023-04-21)
</p>
</div>
</div>



</body>
</html>
